docroot/html/course/view.jsp

```diff
@@ -4,25 +4,82 @@
 	<portlet:param name="mvcPath" value="/html/course/editForm.jsp" />
 </portlet:renderURL>
 
-<div align="right">
-	<liferay-ui:icon image="add_article" message="Add new Course"/>
-	<aui:a href="${addURL}">Add new Course</aui:a>
-</div>
-<liferay-ui:search-container emptyResultsMessage="No result was found">
-	<liferay-ui:search-container-results results="<%= CourseLocalServiceUtil.getCourses(searchContainer.getStart(), searchContainer.getEnd())%>" total="<%= CourseLocalServiceUtil.getCoursesCount() %>" />
-	<liferay-ui:search-container-row
-		className="com.liferay.docs.course.model.Course" modelVar="course" escapedModel="<%= true %>">
-		<liferay-ui:search-container-column-text property="name" />
-		<liferay-ui:search-container-column-text property="description" />
-		<liferay-ui:search-container-column-text property="lecturer"
-			name="Lecturer" />
-		<liferay-ui:search-container-column-text name="status"
-			value='${ course.isStatus() == true ? "Available" : "Unavailable" }' />
-		<liferay-ui:search-container-column-text property="duration" />
-		<liferay-ui:search-container-column-jsp name="Actions"
-			path="/html/course/actionButton.jsp" />
-	</liferay-ui:search-container-row>
-	<liferay-ui:search-iterator />
-
-
-</liferay-ui:search-container>
\ No newline at end of file
+<%
+	PortletURL portletURL = renderResponse.createRenderURL();	
+	String portletURLString = portletURL.toString();	
+	String name = ParamUtil.getString(request, "name");
+	String lecturer = ParamUtil.getString(request, "lecturer");
+	String status = ParamUtil.getString(request, "status");
+%>
+
+<%
+	Map<String, Serializable> attributes = new HashMap<String, Serializable>();
+	attributes.put("name", name);
+	attributes.put("lecturer", lecturer);
+	if(status != "") {
+		attributes.put("status", status);
+	}
+	SearchContext searchContext = SearchContextFactory.getInstance(request);
+	searchContext.setAttributes(attributes);
+	searchContext.setAttribute("paginationType", "more");
+	searchContext.setStart(0);
+	searchContext.setEnd(10);
+	searchContext.setAndSearch(true);
```

Comment:
Avoid using setStart and setEnd statically. When you use pagination, this will cause problems